src/train.py

```diff
@@ -0,0 +1,23 @@
+import os
+os.environ['TF_CPP_MIN_LOG_LEVEL']='3'
```

Comment:
Leave this out, or at least use `.setdefault()`.

Reply:
Done